my-sms/spec/controllers/courses_controller_spec.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CoursesController, type: :controller do
+  let!(:course) { create(:course) }
+
+  describe 'GET #index' do
+    subject { get :index }
+    it 'returns a success response' do
+      should be_success
+      should render_template('index')
```

Comment:
Can you replace `should` with `expect`?

Reply:
Fixed in 44d2330a93064812ea579fda2ed10b625b3cd1e9